entries/s/shipstation.com.json

```
@@ -0,0 +1,12 @@
+{
+  "ShipStation": {
+    "domain": "shipstation.com",
+    "contact": {
+      "facebook": "ShipStation",
+      "twitter": "ShipStation"
```

As given on their Twitter page:
```suggestion
      "twitter": "ShipStationHelp",
      "email": "support@shipstation.com"
```